langs/fr/common.json

```diff
@@ -20,43 +20,43 @@
   "Platforms": "Plateformes",
   "Game List": "Liste des jeux vidéos",
   "More game releases": "Plus de jeux vidéos",
-  "Animation List": "Liste des animations",
+  "Animation List": "Liste des séries et films d'animation",
   "Studios": "Studios",
-  "More animation releases": "Plus d'animations",
-  "Get Lucky": "Soyez chanceux",
+  "More animation releases": "Plus de séries et films d'animation",
+  "Get Lucky": "Je suis chanceux",
   "Random Pull": "Aléatoire",
-  "Soundtrack Count": "Nombres de oundtrack totales",
-  "Highlight Soundtrack": "Soundtrack sélectionnée",
-  "Game Count": "Nombre de soundrack de jeux vidéos",
-  "Animation Count": "Nombre de soundrack d'animations",
-  "Game Soundtracks": "Bande Originale de jeux vidéos",
-  "Animation Soundtracks": "Bande Originale d'animations",
-  "Email_Sent": "An email has been sent to the address linked to the account. Check your spam folder",
-  "Username or email": "Username or email",
-  "Recover password": "Recover password",
-  "Invalid_Login": "Invalid username/email or password",
-  "Username": "Username",
-  "Password": "Password",
-  "Failed_Recover": "Failed to recover password",
-  "Animations": "",
-  "Favorite_Added": "",
-  "Favorite_Removed": "",
-  "Favorite_Add": "",
-  "Favorite_Remove": "",
-  "Favorite_Login": "",
-  "Release Date": "",
-  "Artists": "",
-  "Classification": "",
-  "Game Soundtrack": "",
-  "Animation Soundtrack": "",
-  "Published by": "",
-  "Favorite Score": "",
-  "Check album at": "",
-  "Buy_Original": "",
-  "Download": "",
-  "Related Soundtracks": "",
-  "Become_Donator": "",
-  "Direct": "",
-  "Tracklist": "",
-  "Disc": ""
+  "Soundtrack Count": "Nombre total de pistes",
```

Comment:
It's the number of soundtracks, not the number of tracks, so should be "Nombres de Soundtrack totales"